### ISSUE TITLE
Export an image with Tiff format turns into a jpeg format

### ISSUE THREAD

**Issue by SilvioGrosso** [aferrero2707/PhotoFlow#205]
Hello, 

As discussed on this forum:
https://discuss.pixls.us/t/enhanced-unsharp-mask-in-photoflow-please-test/14540/46

A Tiff format becomes Jpeg when you export an image.
This bug has been confirmed :-)

**Comment by aferrero2707** [comment]
@SilvioGrosso this should be fixed in the current stable branch. Pressing the "Export to..." button now pops up a file chooser dialog whose default file name has an extension that matches the selected file format. Previously ".jpg" was wrongly used in all cases.

Could you please confirm?
Thanks!

**Comment by SilvioGrosso** [comment]
@aferrero2707 

Yep. Now it works fine
Tested on Windows 10 with: 
photoflow-w64-20191109_1805-git-stable-0172654e272c5b270586492435d86ec8b441580e
We can close this bug.

I have also noticed that the GUIs works much better on my small screen (13,3 inches). 
This can be verified by opening more images in the row. Previously, with 2 or more images the bottom part of the GUIs was no longer visible on my small screen. It was fine only with one single image opened.

BTW, now opening and closing many images in succession is much MORE robust. No crash whatsoever in short :-)
I had only one freeze, *not* a real crash while opening and closing many images with PhotoFlow (Windows 10 as usual).
